test/ueberauth/strategy/cognito_test.exs

```diff
@@ -132,11 +180,12 @@ defmodule Ueberauth.Strategy.CognitoTest do
         |> Plug.Conn.fetch_query_params()
         |> Cognito.handle_callback!()
 
-      assert conn.private.cognito_id_token == %{"email" => "foo"}
+      _id_token = conn.private.cognito_id_token
+      assert match?(_id_token, %{"email" => "foo"})
```

Comment:
Can this be assert `%{"email" => "foo"} = conn.private.cognito_id_token`?

And if not, `_id_token` shouldn't have an underscore prefix as it's being used`.